src/Material.cpp

```diff
@@ -816,7 +951,7 @@ void Material::setSigmaS(double* xs, int num_groups_squared) {
 
   for (int dest=0; dest < _num_groups; dest++) {
     for (int orig=0; orig < _num_groups; orig++)
-      _sigma_s[dest*_num_groups+orig] = xs[orig*_num_groups+dest];
+      setSigmaSByGroupInline(xs[orig*_num_groups+dest],orig,dest);
```

Comment:
Was there a ``Material::setSigmaSByGroupInline(...)`` routine in the previous implementation? I don't think so, since it doesn't make much sense to have one (it only made sense to have an inline routine for the getter, but even then, we might do away with it to simplify the code).